### PR TITLE
Reg user nav tg

### PR DESCRIPTION
**Pull request by tyladevon**
Complete Regular User Nav - User Story 6

### CODE REVIEW

File: spec/features/navigation/regular_user_navigation_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      @user = User.create(name: "Dee",
+                         street_address: "4233 Street",
+                         city: "Golden",
+                         state: "CO",
+                         zip: "80042",
+                         email: "deedee@gmail.com",
+                         password: "rainbows1908",
+                         role: 0)
+
+      # session[:user_id] = @user.id
```
Comment (hillstew) [inline spec/features/navigation/regular_user_navigation_spec.rb]: Can we delete this now that we have completed this test?

Comment (rhantak) [reply spec/features/navigation/regular_user_navigation_spec.rb]: Yes!

Comment (hillstew) [inline spec/features/navigation/regular_user_navigation_spec.rb]:
```suggestion
```

File: spec/features/navigation/regular_user_navigation_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+      visit '/'
+
+    end
+
+    it "can see some same links in nav bar" do
+      expect(page).to have_link "All Items"
+      expect(page).to have_link "All Merchants"
+      expect(page).to have_link "Home"
+      expect(page).to have_content("Cart")
```
Comment (hillstew) [inline spec/features/navigation/regular_user_navigation_spec.rb]:
```suggestion
      expect(current_path).to have_content("Cart")
```

File: spec/features/navigation/regular_user_navigation_spec.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    end
+
+    it "can see additional links in nav bar" do
+
+      # save_and_open_page
```
Comment (hillstew) [inline spec/features/navigation/regular_user_navigation_spec.rb]:
```suggestion
```

**Pull request merged**